src/styles/settings/_colors.scss

```diff
@@ -6,69 +6,64 @@
  ========================================================================== */
 
 $color-gray-1000: #000000;
-$color-gray-900: #0F171D;
-$color-gray-800: #1D2B35;
-$color-gray-700: #2B414F;
+$color-gray-900: #0f171d;
+$color-gray-800: #1d2b35;
+$color-gray-700: #2b414f;
 $color-gray-600: #385265;
-$color-gray-500: #45647B;
+$color-gray-500: #45647b;
 $color-gray-400: #557891;
-$color-gray-300: #7894A7;
-$color-gray-200: #9CAFBD;
-$color-gray-150: #BAC7D1;
-$color-gray-100: #D6DEE3;
-$color-gray-50: #EDEFF2;
+$color-gray-300: #7894a7;
+$color-gray-200: #9cafbd;
+$color-gray-150: #bac7d1;
+$color-gray-100: #d6dee3;
+$color-gray-50: #edeff2;
 
 /* Primary colors
    ========================================================================== */
 
-$color-night-blue: #3A586F;
-$color-dark-night-blue: #182F42;
-$color-teal-bright: #25A4CF;
-$color-teal: #1D637A;
+$color-night-blue: #3a586f;
+$color-dark-night-blue: #182f42;
+$color-teal-bright: #25a4cf;
+$color-teal: #1d637a;
 $color-teal-dark: #124455;
 
 /* Secondary colors
    ========================================================================== */
 
-$color-error-200: #EE8176;
-$color-green: #46D46E;
-$color-blue: #4D76F3;
-$color-soft-blue: #7CAACF;
-
-/* Secondary colors
-   ========================================================================== */
-
-$color-ocean-green: #57C2A0;
-$color-silver: #DBDBDB;
-$color-tapestry: #B75B99;
-$color-wine-red: #98485C;
-$color-night-blue: #3A586F;
-$color-yellow: #F3AA2E;
-$color-green: #9DB410;
```

Comment:
wouldn't this be the correct green color instead of #46D46E, since the correct blue color also came from this block of colors?